crates/starknet/src/account.rs

```diff
@@ -37,7 +37,7 @@ impl Account {
         class_hash: ClassHash,
         contract_class: ContractClass,
         fee_token_address: ContractAddress,
-    ) -> DevnetResult<Self> {
```

Comment:
I guess we would ideally change all occurrences of `DevnetResult`? Maybe we can delegate this to a separate issue?

Reply:
Yeap, I would prefer to move it into error.rs

Reply:
But to much changes for this pr then

Reply:
Create an issue